tailcall-prettier/src/prettier.rs

```diff
@@ -71,7 +71,11 @@ impl Prettier {
                     child = child.arg("--config").arg(config);
                 }
 
-                let mut child = child.stdin(Stdio::piped()).stdout(Stdio::piped()).spawn()?;
+                let mut child = child
+                    .stdin(Stdio::piped())
+                    .stdout(Stdio::piped())
+                    .spawn()
+                    .map_err(|_| Error::PrettierNotFound)?;
```

Comment:
```suggestion
                    .map_err(|e| Error::System(e))?;
```
- Shouldn't loose the cause of the error
- Could rename this to something that signals that the error is caused because of a system issue.

Reply:
the main issue that std error is not helpful at all. 

I've updated the code to store other kind of errors except the not found to provide additional info